source/frontends/libretro/game.cpp

```diff
@@ -4,38 +4,17 @@
 #include "frontends/libretro/retroframe.h"
 #include "frontends/libretro/rkeyboard.h"
 #include "frontends/common2/utils.h"
-#include "frontends/common2/ptreeregistry.h"
-#include "frontends/common2/programoptions.h"
 
 #include "Common.h"
 #include "Interface.h"
 
 #include "linux/keyboardbuffer.h"
 #include "linux/paddle.h"
 #include "linux/context.h"
+#include "linux/registryclass.h"
 
 #include "libretro.h"
 
-#define APPLEWIN_RETRO_CONF "/tmp/applewin.retro.conf"
-
-namespace
-{
-
-  void saveRegistryToINI(const std::shared_ptr<common2::PTreeRegistry> & registry)
```

Comment:
libretro cores should not write files, and i couldn't come up with a good reason to keep this. the frontend will store the modified configuration according to its implementation.

Reply:
This is a feature I use to debug issues, it saves its configuration to a file I can reload in a non-liretro run.

Reply:
This is my previous attempt: https://github.com/audetto/AppleWin/commit/e7795ab0d0546cf49529785499852efb2ef85346

I will review the whole thing again and extend it to all frontends.

Reply:
Does that mean this is okay, or you want me to put it back?